Sources/AccessibilitySnapshot/Core/Swift/Classes/UIAccessibility+SnapshotAdditions.swift

```diff
@@ -479,6 +505,21 @@ extension NSObject {
                 comment: "Description of the last element in a landmark container",
                 locale: locale
             )
+            self.textEntryTraitName = "Text Field.".localized(
+                key: "trait.text_field.description",
+                comment: "Description for the 'text entry' accessibility trait",
+                locale: locale
+            )
+            self.textEntryTraitHint = "Double tap to edit.".localized(
+                key: "trait.text_field.hint",
+                comment: "Hint describing how to use elements with the 'text entry' accessibility trait",
+                locale: locale
+            )
+            self.scrollableTextEntryTraitHint = "Double tap to edit., Use the rotor to access Misspelled Words".localized(
+                key: "trait.scrollable_text_field.hint",
+                comment: "Hint describing how to use elements with the 'text entry' and 'scrollable' accessibility traits",
+                locale: locale
+            )
```

Comment:
Do y'all have a process for getting the translated strings?

Reply:
Since we're trying to match VoiceOver exactly, the best way is to run the app with VoiceOver in each of the supported languages and copy down the string it outputs. I just opened #203 to make this a bit easier, with schemes forcing the app into each of the supported languages.

Reply:
I usually use Google Translate first to get it close, then verify with VoiceOver and tweak as needed. Little bit easier than trying to write it out from scratch. 😄

Reply:
I've added the German and Russian translations.